### PR TITLE
[WIP] generate (verbose) ascii for initConsensusPoset plays array using...

### PR DESCRIPTION
**Pull request by dev10**
[WIP] generate (verbose) ascii for initConsensusPoset plays array using gographviz and graph-easy perl script
This requires the graph-easy perl script to be installed: http://www.bloodgate.com/perl/graph/manual/

Please check if what you want to add to `go-lachesis` list meets [quality standards](https://github.com/Fantom-foundation/go-lachesis/blob/master/CONTRIBUTING.md#quality-standard) before sending pull request.

**Please provide package links to:**

- github.com repo:
- godoc.org:
- goreportcard.com:
- coverage service link ([cover.run](https://cover.run/), [gocover](http://gocover.io/), [coveralls](https://coveralls.io/) etc.), example: `[cover.run](https://cover.run/go?tag=golang-1.10&repo=github.com%2Fuser%2Frepository)`

**Make sure that you've checked the boxes below before you submit PR:**
- [x] I have added my package in alphabetical order.
- [x] I have an appropriate description with correct grammar.
- [x] I know that this package was not listed before.
- [x] I have added godoc link to the repo and to my pull request.
- [x] I have added coverage service link to the repo and to my pull request.
- [x] I have added goreportcard link to the repo and to my pull request.
- [x] I have read [Contribution guidelines](https://github.com/Fantom-foundation/go-lachesis/blob/master/CONTRIBUTING.md#contribution-guidelines), [maintainers note](https://github.com/Fantom-foundation/go-lachesis/blob/master/CONTRIBUTING.md#maintainers) and [Quality standard](https://github.com/Fantom-foundation/go-lachesis/blob/master/CONTRIBUTING.md#quality-standard).

### CODE REVIEW

**Comment by devintegral** [comment]
@dev10, I'd be very grateful if you would make more tidy PR.  Why don't you remove template from PR description? Or why don't you check for simplest errors:
```
> go test .
poset_test.go:13:2: build constraints exclude all Go files in /home/fabel/Work/src/github.com/Fantom-foundation/go-lachesis/src/common/testing/ascii_generator
```

File: src/poset/poset_test.go [hunk 1]
```diff
@@ -1407,63 +1427,181 @@ func TestInsertEventsWithBlockSignatures(t *testing.T) {
...
-	|	|   | \ |
-	|   |   |  f2b
-	|	|   |   |
-	----------<f2>------------------------------
-		|   | / |  Round 0
-		|  e10  |
-	    | / |   |
-	   [e0][e1][e2]
-		0   1    2
+                 +-----------------------------------------------------+
```
Comment (devintegral) [inline src/poset/poset_test.go]: In my opinion, previous ascii-grafics are more readable than new.

Comment (SamuelMarks) [reply src/poset/poset_test.go]: @devintegral Yeah, so I didn't know you were working on it also. I asked @dev10 to work on this earlier, and we were sharing code + output snippets in Slack. Got Peter to open this thread for Maxim to review.

We will need bidirectional generation, from ASCII graphic to graphviz and back again, in addition to going from the graph in Go to a graphviz DOT, then back from DOT to Go.

Comment (devintegral) [reply src/poset/poset_test.go]: @SamuelMarks, how do you propose to use translation from graphviz to ASCII graphic? I can't imagine real case you are about.

**Comment by devintegral** [comment]
outdated

**Pull request closed without merging**